### DISCUSSION TITLE
Arabic bert model

### DISCUSSION THREAD

**Discussion by AyaElsawyElghaysh** [aub-mind/arabert#105] [Q&A]
How can i use that model on my dataset?

**Comment by WissamAntoun** [comment]
check the examples folder

**Reply by AyaElsawyElghaysh** [reply]
thank you .

**Reply by AyaElsawyElghaysh** [reply]
if i use this colab file https://colab.research.google.com/drive/1hik0L_Dxg6WwJFcDPP1v74motSkst4gE?usp=sharing#scrollTo=m-KYmncJY0Vs
and create my datasets and pass its path Is it working?

**Reply by WissamAntoun** [reply]
You might have to change some code in the dataset loader

**Reply by AyaElsawyElghaysh** [reply]
ok thank you
